staker/validatorwallet/contract.go

```diff
@@ -179,6 +183,37 @@ func (v *Contract) executeTransaction(ctx context.Context, tx *types.Transaction
 	return v.dataPoster.PostSimpleTransaction(ctx, auth.Nonce.Uint64(), *v.Address(), data, gas, auth.Value)
 }
 
+// Exposed for tests
+func (v *Contract) CreateWalletContract(
+	ctx context.Context,
+	validatorWalletFactoryAddr common.Address,
+) (*types.Transaction, error) {
+	var initialExecutorAllowedDests []common.Address
+	txData, err := validatorWalletCreatorABI.Pack("createWallet", initialExecutorAllowedDests)
+	if err != nil {
+		return nil, err
+	}
+
+	auth, err := v.getAuth(ctx, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	gas, err := gasForTxData(
+		ctx,
+		v.l1Reader,
+		auth,
+		&validatorWalletFactoryAddr,
+		txData,
+		v.getExtraGas,
+	)
+	if err != nil {
+		return nil, fmt.Errorf("getting gas for tx data: %w", err)
+	}
+
+	return v.dataPoster.PostSimpleTransaction(ctx, auth.Nonce.Uint64(), validatorWalletFactoryAddr, txData, gas, auth.Value)
```

Comment:
`auth.Value` will be nil here. It might not cause any issues in the data poster right now, but I'd like to either explicitly sanitize it in the data poster's PostTransaction method or just use common.Big0 here.

Reply:
Done, using common.Big0.